debbuild

```diff
@@ -245,22 +276,55 @@ sub load_static_config {
 sub load_config {
   # Load user configuration, permitting local override
   my $homedir = $ENV{HOME} // $ENV{LOGDIR} // (getpwuid($<))[7];
-  foreach my $macros ( ("$static_config{debconfigdir}/macros",
-                        glob("$static_config{debconfigdir}/macros.d/macros.*"),
-                        "$static_config{sysconfdir}/debbuild/macros",
-                        glob("$static_config{sysconfdir}/debbuild/macros.*"),
-                        "$homedir/.debmacros") ) {
+  vdebug("Loading ...", 3, "load_config");
+  my @cfgs = ("$static_config{debconfigdir}/macros",
+              glob("$static_config{debconfigdir}/macros.d/macros.*"),
+              "$static_config{sysconfdir}/rpm/macros",
+              glob("$static_config{sysconfdir}/rpm/macros.*"),
+              "$static_config{sysconfdir}/debbuild/macros",
+              glob("$static_config{sysconfdir}/debbuild/macros.*"),
+              "$homedir/.rpmmacros",
+              "$homedir/.debmacros");
```

Comment:
We _intentionally_ do not read RPM macro paths by default. Please remove this.

Reply:
I put it there just because a number of package just put the macroses there, for such a case the only possible way other than that is to rebuild the package providing rpm macro. But I'm trying to avoid rebuilding packages just to move macroses to the different location.

Reply:
Yeah, but RPM and Debian systems often need macro adaptations for other reasons, so reading RPM paths is a *bad* idea because it'll do the wrong thing. This is why the `debbuild-macros` project exists, to provide those adapted versions for Debian systems.

Reply:
Yes, I know it, that's why the order it loads the macro files is set to load all the debuild related config later than rpm. In this case it will redefine the macroses from debmacros configs. I've tested it for a number of different cases and I'm sure this way is the best at least for our case.

Reply:
@Conan-Kudo we need that in Uyuni to be able to build Salt for Debian and Ubuntu. Look at this project config (and subprojects) and cry :-) 
https://build.opensuse.org/projects/systemsmanagement:saltstack:bundle/prjconf

Reply:
I am not convinced that it's a good idea to allow this, especially since you have the ability to build your own macros packages and already had to do that to build it for Debian/Ubuntu targets.

Reply:
For example, `debbuild` is packaged in Fedora to let people build Debian packages from Fedora. And having these paths in there would completely break its behaviors on that system.

Reply:
Hi Neal.
I removed the changes related to rpmmacro files and Maintainer. Are you ok with these changes?
In our projects it's not an option to maintain a list of additional packages for just a copy of macroses for debbuild. One of the goals in this project was to avoid the modification is specs and macroses as much as possible to reduce any possible entities to maintain.

Reply:
Yeah, I understand that. But another solution would be to submit changes to those packages in OBS to integrate what you need.

Reply:
Yes, but in this case it's exact extra entities to maintain. Another goal was to make `debbuild` to do marco expanding the same way as `rpm` does. That's why I mentioned the difference with `dash`. Substitution is one of items performed different way and some syntax difference for causing fails on processing `%prep` and `%build`

Reply:
Technically, even openSUSE is trying to not use bash with `rpm` (with the whole busybox `sh` thing...), and both `dash` and busybox `sh` are similar (since they're both `ash` variants).